lib/fathead/python_six/output.txt

```diff
@@ -0,0 +1,142 @@
+string or a code object.  If globals or locals are not given, they will
+default to the scope of the caller.  If just globals is given, it will also
+be used as locals.</p><pre><code>\nNote\nPython 3’s exec() doesn’t take keyword arguments, so calling\nexec() with them should be avoided.\n</code></pre></section>	https://pythonhosted.org/six//index.html
+print_	A										<section class="prog__container"><p>Print args into file.  Each argument will be separated with sep and
+end will be written to the file after the last argument is printed.  If
+flush is true, file.flush() will be called after all data is written.</p><pre><code>\nNote\nIn Python 2, this function imitates Python 3’s print() by not\nhaving softspace support.  If you don’t know what that is, you’re probably\nok. :)\n</code></pre></section>	https://pythonhosted.org/six//index.html
+raise_from	A										<section class="prog__container"><p>Raise an exception from a context.  On Python 3, this is equivalent to
+raise exc_value from exc_value_from.  On Python 2, which does not support
+exception chaining, it is equivalent to raise exc_value.</p></section>	https://pythonhosted.org/six//index.html
+reraise	A										<section class="prog__container"><p>Reraise an exception, possibly with a different traceback.  In the simple
+case, reraise(*sys.exc_info()) with an active exception (in an except
+block) reraises the current exception with the last traceback.  A different
+traceback can be specified with the exc_traceback parameter.  Note that
+since the exception reraising is done within the reraise() function,
+Python will attach the call frame of reraise() to whatever traceback is
+raised.</p></section>	https://pythonhosted.org/six//index.html
+with_metaclass	A										<section class="prog__container"><p>Create a new class with base classes bases and metaclass metaclass.  This
+is designed to be used in class declarations like this:</p><pre><code>from six import with_metaclass\n\nclass Meta(type):\n    pass\n\nclass Base(object):\n    pass\n\nclass MyClass(with_metaclass(Meta, Base)):\n    pass\n\n</code></pre></section>	https://pythonhosted.org/six//index.html
+add_metaclass	A										<section class="prog__container"><p>Class decorator that replaces a normally-constructed class with a
+metaclass-constructed one.  Example usage:</p><pre><code>@add_metaclass(Meta)\nclass MyClass(object):\n    pass\n\n</code></pre></section>	https://pythonhosted.org/six//index.html
+b	A										<section class="prog__container"><p>A “fake” bytes literal.  data should always be a normal string literal.  In
+Python 2, b() returns a 8-bit string.  In Python 3, data is encoded
+with the latin-1 encoding to bytes.</p><pre><code>\nNote\nSince all Python versions 2.6 and after support the b prefix,\nb(), code without 2.5 support doesn’t need b().\n</code></pre></section>	https://pythonhosted.org/six//index.html
```

Comment:
@iamrajhans let's remove trailing `\n` We don't need an `\n` at the end of a sentence 👍